orbotservice/src/main/java/org/torproject/android/service/OrbotService.java

```diff
@@ -223,64 +225,45 @@ private void createNotificationChannel() {
         mNotificationManager.createNotificationChannel(mChannel);
     }
 
-    @SuppressLint("NewApi")
+    @SuppressLint({"NewApi", "RestrictedApi"})
```

Comment:
This `RestrictedApi` thing is the call to `mNotifyBuilder.mActions.clear()` which IMO is fine to do. Normally you wouldn't need to clear actions because you wouldn't hang on to a notify builder object like we do here, but anyway ya I think it's fine tbh.